remote_attestation/rust/src/crypto.rs

```diff
@@ -114,6 +113,19 @@ impl AeadEncryptor {
             .map_err(|error| anyhow!("Couldn't decrypt data: {:?}", error))?;
         Ok(decrypted_data.to_vec())
     }
+
+    /// Generate a random nonce.
+    /// `ring::aead` uses 96-bit (12-byte) nonces.
+    /// https://briansmith.org/rustdoc/ring/aead/constant.NONCE_LEN.html
+    fn generate_nonce() -> [u8; aead::NONCE_LEN] {
+        let mut rng = rand::thread_rng();
```

Comment:
Is it intentional to use `rand::thread_rng()` here and `ring::rand::SystemRandom` below?

Reply:
Changed everything to `ring::rand::SystemRandom`